app/full_text_search.py

```diff
@@ -209,10 +252,10 @@ def create_romanized_name_fields(romanize_method, **kwargs):
     romanized_names_list = []
     romanize_method_name = romanize_method.__name__
     for field in kwargs:
-        romanized_name = romanize_method(kwargs[field])
-        if romanized_name:
+        romanized_names = romanize_method(kwargs[field])
+        for romanized_name in romanized_names:
             fields.extend(create_fields_for_rank(field, romanized_name))
```

Comment:
Doesn't this create_fields_for_rank create multiple fields with the same name if romanized_names has more than one items?